internal/pkg/template/templates/environment/cf.yml

```diff
@@ -585,14 +585,20 @@ Resources:
 {{include "ar-vpc-connector" . | indent 2}}
 {{- end}}
 {{- if .VPCConfig.FlowLogs }}
+  LogGroup:
+    Type: AWS::Logs::LogGroup
+    Metadata:
+      'aws:copilot:description': 'A  log group to create a flow log for the VPC'
```

Comment:
```suggestion
      'aws:copilot:description': 'A log group to create a flow log for the VPC'
```

Reply:
Another suggestion could be -
"A CloudWatch log group to hold your VPC flow logs"